src/com/owncloud/android/AppRater.java

```diff
@@ -0,0 +1,77 @@
+/**
+ *   ownCloud Android client application
+ *
+ *   Copyright (C) 2016 ownCloud GmbH.
+ *
+ *   This program is free software: you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License version 2,
+ *   as published by the Free Software Foundation.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+package com.owncloud.android;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.support.v4.app.FragmentActivity;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentTransaction;
+
+import com.owncloud.android.ui.dialog.RateMeDialog;
+
+public class AppRater {
+
+    private static final String DIALOG_RATE_ME_TAG = "DIALOG_RATE_ME";
+
+    private final static int DAYS_UNTIL_PROMPT = 2;
+    private final static int LAUNCHES_UNTIL_PROMPT = 2;
+    private final static int DAYS_UNTIL_NEUTRAL_CLICK = 1;
+
+    public static void appLaunched(Context mContext, String packageName) {
+        SharedPreferences prefs = mContext.getSharedPreferences("app_rater", 0);
```

Comment:
The preferences names (app_rater, date_neutral, date_first_launch, don't_show_again) should go in constants in the top of the class